lib/CL/devices/tce/tce_common.cc

```diff
@@ -64,6 +64,10 @@ using namespace TTAMachine;
 
 //#define DEBUG_TTA_DRIVER
 
+#define STRINGIFY(x) #x
+#define TOSTRING(x) STRINGIFY(x)
+#define TCECC_PATH TOSTRING(TCECC)
```

Comment:
I think I'd prefer having this handled by a `#cmakedefine` in one of the config headers. And maybe call it TCECC_EXECUTABLE or something, since from the usage it seems to refer to the actual binary, not just a toolchain directory.

Reply:
And maybe convert the naming (at least gradually) to openasip / oacc.

Reply:
Thanks! Moved this to config.h.in
Also changed the name to OACC_EXECUTABLE